tooling/lsp/src/requests/goto_declaration.rs

```diff
@@ -32,49 +33,32 @@ fn on_goto_definition_inner(
     let workspace = resolve_workspace_for_source_path(file_path.as_path()).unwrap();
     let package = workspace.members.first().unwrap();
 
-    let package_root_path: String = package.root_dir.as_os_str().to_string_lossy().into();
-
     let mut workspace_file_manager = file_manager_with_stdlib(&workspace.root_dir);
     insert_all_files_for_workspace_into_file_manager(&workspace, &mut workspace_file_manager);
     let parsed_files = parse_diff(&workspace_file_manager, state);
 
     let (mut context, crate_id) =
         nargo::prepare_package(&workspace_file_manager, &parsed_files, package);
 
-    let interner;
-    if let Some(def_interner) = state.cached_definitions.get(&package_root_path) {
-        interner = def_interner;
+    let package_root_path: String = package.root_dir.as_os_str().to_string_lossy().into();
+    let interner = if let Some(def_interner) = state.cached_definitions.get(&package_root_path) {
+        def_interner
```

Comment:
```suggestion
    let package_root_path = package.root_dir.as_os_str().to_string_lossy().into_owned();
    let interner = if let Some(def_interner) = state.cached_definitions.get(&package_root_path) {
        def_interner
```
Seems like this if let block is repeated in the next file as well - maybe it also needs a helper